app/helpers/cms_helper.rb

```diff
@@ -192,4 +192,8 @@ def get_resource(fragment_text, fragment_link, button_text, button_class)
       false
     end
   end
+
+  def cms_fragment_content_datetime(identifier, page)
+    page.fragments.find_by(tag: 'date_not_null', identifier: identifier).datetime
```

Comment:
@stanleypliu When possible, it's always a good idea to prevent possible exceptions and return a fallback value.
In this case for example, even though it's not very likely, it might happen that you can't find a fragment with that tag and identifier; so if that find returns `nil` your code will throw an exception with `datetime` because it can't me invoked on nil. So you could make use of `try`, like: `find_by(...).try(:datetime)` which invokes the method if the previous result is not nil, returns nil otherwise.